addons/godot_tours/core/editor_interface_access.gd

```diff
@@ -310,14 +346,17 @@ true).get_child(1, true)
 	tabs = [main_screen_tabs, scene_tabs, filesystem_tabs, inspector_tabs]
 	controls_maps = [
```

Comment:
As we talked about on discord, could you see if you can remove the need for mapping tabs and controls? This controls map is never exhaustive and as I was working on tours, I found myself having to add things to it manually (or could've been programmatically) and I see it's going to keep taking time when creating content, so I'd like to remove the need for this.

Reply:
NB: it's used in the bubble script too iirc but there also I think we may be able to work without this. The bubble connected to the draw signals of many controls to move/redraw when things were resized in the editor. If we prevent resizing, then we don't need this as much.